licensed_pile/licenses.py

```diff
@@ -24,7 +29,16 @@ class PermissiveLicenses(StringEnum):
     GFDL = "GNU Free Documentation License"
     APACHE_2 = "Apache 2 License - https://www.apache.org/licenses/LICENSE-2.0"
     MIT = "MIT License"
-    BSD = "BSD License"
```

Comment:
Is there any source in here already that was BSD licensed? If so we should make sure to update it to use one of the BSD_2 or BSD_3 to avoid breakage

Reply:
I don't think any code datasources have been checked in yet so I don't think anyone would have used it.